### PR TITLE
Update darkmode functionality

### LINKED ISSUE #568: Dark Mode update
- When the "Back up seed phrase" button is clicked on wallet page, "Keep in mind" page shows up, but the text is unreadable in dark mode:

- On tickets page, when the "Purchase ticket" button is clicked, a modal shows up which has unreadable text and border lines that match the background color:

- On the modal displayed above, when the "Select VSR" field is clicked, another modal shows up which contains the different VSRs. This modal has unreadable textin dark mode as shown below:

- godcr uses incorrect logo in dark mode

The UI should switch to using

this icon instead

### LINKED ISSUE #634: Update Dark Mode Icon
godcr uses incorrect logo in dark mode

Screen Shot 2021-09-20 at 9 38 39 AM

The UI should switch to using 

godcr-master-icon 

this icon instead

Somewhat related to [this issue](https://github.com/planetdecred/godcr/issues/568)

### PR DESCRIPTION
**Pull request by Sirmorrison**
This pr updates darkmode feature.
fix #568

### CODE REVIEW

File: ui/page/send/utxo_page.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func (pg *UTXOPage) Handle() {
 			}
 			icoBtn := pg.Theme.IconButton(decredmaterial.MustIcon(widget.NewIcon(icons.ContentContentCopy)))
 			icoBtn.Inset, icoBtn.Size = layout.UniformInset(values.MarginPadding5), values.MarginPadding20
-			icoBtn.Background = pg.Theme.Color.LightGray
+			icoBtn.ChangeColorStyle(&values.ColorStyle{Background: pg.Theme.Color.Gray4}) // TODO: Set foreground color too?
```
Comment (devchoplife) [inline ui/page/send/utxo_page.go]: Are we setting foreground color here??

File: ui/util.go [hunk 2]
```diff
@@ -38,10 +37,6 @@ func editorsNotEmpty(editors ...*widget.Editor) bool {
 	return true
 }
 
-func generateRandomNumber() int {
-	return rand.New(rand.NewSource(time.Now().UnixNano())).Int()
-}
```
Comment (devchoplife) [inline ui/util.go:43]: Remove GenerateRandomNumber function on ui/page/utils.go too. 
Here https://github.com/planetdecred/godcr/blob/01988a89c4ed254cd1403a64e0f8fcd95443881d/ui/page/utils.go#L40

**Review by devchoplife — DISMISSED** [review]
Looks good and works fine across all pages

**Review by dreacot — COMMENTED** [review]
Stellar work was done on this PR, there are certain dark mode issues that still persist, i'm of the opinion that this PR gets merged regardless and issues found are broken down and tackled in bits in order not to unnecessarily delay having these changes in master

1. The specific peer text needs adjustment
Screenshot 2021-12-06 at 8 37 11 PM

2. the wallet expand icon, and the arrow icon on the backup card needs to be modified
Screenshot 2021-12-06 at 8 39 55 PM

3. the rewards earned value isn't visible
Screenshot 2021-12-06 at 8 41 47 PM

4. the expired icon needs to be modified
Screenshot 2021-12-06 at 8 42 10 PM

5. the hover colour needs to be adjusted
Screenshot 2021-12-06 at 8 42 42 PM

6. the refresh icon on the proposals page needs to be adjusted
Screenshot 2021-12-06 at 8 43 15 PM

7. the view on politeia icon needs to be modified
Screenshot 2021-12-06 at 8 43 34 PM

8. hovering on the scrollbar produces a colour that's too dark
Screenshot 2021-12-06 at 8 48 36 PM

9. the max buttons need to be consistent, one has a white text, one has a black text, also, the arrow in-between both input boxes need work
Screenshot 2021-12-06 at 8 49 39 PM

10.  the icons on the more page needs to be modifed
Screenshot 2021-12-06 at 8 50 13 PM

11. this modal needs work, firstly the input box for the number of tickets should be a different shade of black
Screenshot 2021-12-06 at 8 51 06 PM

there are other minor issues like this lurking around

File: ui/page/start_page.go [hunk 3]
```diff
@@ -178,14 +167,26 @@ func (sp *startPage) loadingSection(gtx layout.Context) layout.Dimensions {
 					})
 				}),
 				layout.Rigid(func(gtx layout.Context) layout.Dimensions {
-					return sp.networkType.Layout(gtx)
+					netType := sp.WL.Wallet.Net
+					if sp.WL.Wallet.Net == "testnet3" {
```
Comment (dreacot) [inline ui/page/start_page.go]: `testnet3` this should be a constant as it's used in more than one place

**Review by dreacot — CHANGES_REQUESTED** [review]

**Comment by Sirmorrison** [comment]
> Stellar work was done on this PR, there are certain dark mode issues that still persist, i'm of the opinion that this PR gets merged regardless and issues found are broken down and tackled in bits in order not to unnecessarily delay having these changes in master
> 
> 1. The specific peer text needs adjustment
> 
> Screenshot 2021-12-06 at 8 37 11 PM
> 
> 2. the wallet expand icon, and the arrow icon on the backup card needs to be modified
> 
> Screenshot 2021-12-06 at 8 39 55 PM
> 
> 3. the rewards earned value isn't visible
> 
> Screenshot 2021-12-06 at 8 41 47 PM
> 
> 4. the expired icon needs to be modified
> 
> Screenshot 2021-12-06 at 8 42 10 PM
> 
> 5. the hover colour needs to be adjusted
> 
> Screenshot 2021-12-06 at 8 42 42 PM
> 
> 6. the refresh icon on the proposals page needs to be adjusted
> 
> Screenshot 2021-12-06 at 8 43 15 PM
> 
> 7. the view on politeia icon needs to be modified
> 
> Screenshot 2021-12-06 at 8 43 34 PM
> 
> 8. hovering on the scrollbar produces a colour that's too dark
> 
> Screenshot 2021-12-06 at 8 48 36 PM
> 
> 9. the max buttons need to be consistent, one has a white text, one has a black text, also, the arrow in-between both input boxes need work
> 
> Screenshot 2021-12-06 at 8 49 39 PM
> 
> 10. the icons on the more page needs to be modifed
> 
> Screenshot 2021-12-06 at 8 50 13 PM
> 
> 11. this modal needs work, firstly the input box for the number of tickets should be a different shade of black
> 
> Screenshot 2021-12-06 at 8 51 06 PM
> 
> there are other minor issues like this lurking around

Most of the icons here are not widget icons but icons from the mock up. The dark mode versions are not yet ready as this is not a priority design ATM.

File: ui/page/start_page.go (hunk 3 above)
Comment (Sirmorrison) [reply ui/page/start_page.go]: The display text should be `testnet` not `testnet3` as `testnet3` is the name of the backend folder.

**Comment by Sirmorrison** [comment]
I will make modifications to the font text and 
widget icon color

File: ui/page/start_page.go (hunk 3 above)
Comment (dreacot) [reply ui/page/start_page.go]: I meant to say the if statement should check for something like `if sp.WL.Wallet.Net == Constants.TESNET3`

**Review by dreacot — APPROVED** [review]

**Pull request merged**